tee-worker/ts-tests/README.md

```diff
@@ -4,40 +4,43 @@ ts-tests of tee-worker
 
 ## Environment
 
-Node versions 16.x or 18.x, yarn v3
+-   [ ] TODO add docs re nvm, corepack, etc.
 
-## Installation
+[nvm](https://github.com/nvm-sh/nvm) should be installed
 
-`cd tee-worker/ts-tests`
+## Installation
 
-`yarn`
+```
+cd tee-worker/ts-tests
+corepack yarn
+```
 
 ## Type Generated
 
-Update parachain metadata: `yarn workspace parachain-api update-metadata` (requires the parachain is running)
+Update parachain metadata: `corepack yarn workspace parachain-api update-metadata` (requires the parachain is running)
 
-Update sidechain metadata: `yarn workspace sidechain-api update-metadata` (requires the worker is running)
+Update sidechain metadata: `corepack yarn workspace sidechain-api update-metadata` (requires the worker is running)
 
-Generate parachain type: `yarn workspace parachain-api build`
+Generate parachain type: `corepack yarn workspace parachain-api build`
```

Comment:
This still doesn't work for me.

```
tee-worker/ts-tests$ nvm list
       v14.20.0
       v16.20.1
->      v20.1.0
         system
default -> node (-> v20.1.0)
iojs -> N/A (default)
unstable -> N/A (default)
node -> stable (-> v20.1.0) (default)
stable -> 20.1 (-> v20.1.0) (default)
lts/* -> lts/hydrogen (-> N/A)
lts/argon -> v4.9.1 (-> N/A)
lts/boron -> v6.17.1 (-> N/A)
lts/carbon -> v8.17.0 (-> N/A)
lts/dubnium -> v10.24.1 (-> N/A)
lts/erbium -> v12.22.12 (-> N/A)
lts/fermium -> v14.21.3 (-> N/A)
lts/gallium -> v16.20.1
lts/hydrogen -> v18.16.1 (-> N/A)
```

```
 node --version
v20.1.0
```
```
corepack yarn --version
3.6.1
```

Looks like only `yarn` version is locked for me. Did I miss anything ?

Reply:
Yeah; I haven't updated the instructions yet to tell everyone to `nvm use` :D
Need to figure out where exactly; I'm thinking at least the `tee-worker/ts-tests/README.md` but probably higher up as well (top-level `README.md`? Not sure).

There are solutions out there to update the node version automatically upon entering the directory; they still require some hassle to set up locally tho :sweat_smile:

Reply:
>tee-worker/ts-tests/README.md but probably higher up as well (top-level README.md? Not sure.

I feel it's more suitable to update the README in the 'ts-tests' directory. The focus top-level is still more on the parachain. :)

Reply:
True, but the top level README does have a section about running the ts-tests so I had to at least mention it there :D

Reply:
Update: works as expected with `nvm use`